### PR TITLE
Added support for no default date

### PR DESCRIPTION
**Pull request by amarsyla**
Default_date option now accepts boolean values. If set to true, the current date will be parsed. If set to no, no default date will be set when initializing the plugin.

### CODE REVIEW

File: js/jquery.pickmeup.js [hunk 1]
```diff
@@ -728,7 +728,7 @@
 			options.binded.fill();
 			if ($this.is('input')) {
 				$this
-					.pickmeup('set_date', parseDate($this.val() ? $this.val() : options.default_date, options.format, options.separator, options.locale))
+					.pickmeup('set_date', parseDate($this.val() ? $this.val() : ((options.default_date === true) ? new Date : options.default_date), options.format, options.separator, options.locale))
```
Comment (nazar-pc) [inline js/jquery.pickmeup.js]: I've merged it locally and simplifying some things, I just want to ask why did you add `true` support here? It is not mentioned in modified readme, also since `options.default_date` is `new Date` just by default, why do we need it at all? Looks like might be ignored, right?

**Comment by amarsyla** [comment]
In the first pull request I did, I added support only for false, but when I tried setting the value to true, I noticed that the plugin stopped working. Then, I updated it in this pull request. So, just in case any user tries to set it to true, today's date will be returned instead of an error.

**Comment by nazar-pc** [comment]
OK, I've just modified readme to `false` instead of `boolean` to make it clear.
Also, you don't need to close pull request and open new one if you want to add commit - just add commit to the same branch - it will appear here automatically.

**Comment by amarsyla** [comment]
Okay, thanks for the advice. I'm new to Git and still learning this stuff.

**Comment by nazar-pc** [comment]
I've picked only first commit and modified things a bit: https://github.com/nazar-pc/PickMeUp/commits/master
New release 2.8.0 already available, let me know if something will go wrong and thanks for contribution!

**Comment by amarsyla** [comment]
No problem, thanks for the credit. Also, how do you minify the JS files? I used UglifyJS, but it rearranged the code, which thing I don't like.

**Comment by nazar-pc** [comment]
I have no automation for this project yet, I'm doing it from IDE PhpStorm: Right click on file > Refactor > Compress
